configure.in

```diff
@@ -908,14 +909,14 @@ AC_SUBST([WITH_PLUGINS])
 # -O2 implies -fstrict-aliasing, but the code is not safe for that
 #
 AC_MSG_CHECKING([whether $CC supports -fno-strict-aliasing])
-OLD_CFLAGS="$CFLAGS"
-CFLAGS="$CFLAGS -fno-strict-aliasing"
+OLD_CPPFLAGS="$CPPFLAGS"
+CPPFLAGS="$CPPFLAGS -fno-strict-aliasing"
```

Comment:
Shouldn't we keep the no strict aliasing CFLAG until we fully migrated?

Reply:
It' kept. I just made that flag available for .c and .cpp files.